packages/gatsby-plugin-mdx/utils/gen-mdx.js

```diff
@@ -142,47 +139,20 @@ ${code}`
 
   if (!isLoader) {
     debug(`compiling scope`)
-    const instance = new BabelPluginPluckImports()
-    const result = babel.transform(code, {
-      configFile: false,
-      plugins: [
-        instance.plugin,
-        objRestSpread,
-        htmlAttrToJSXAttr,
-        removeExportKeywords,
-      ],
-      presets: [
-        require(`@babel/preset-react`),
-        [
-          require(`@babel/preset-env`),
-          {
-            useBuiltIns: `entry`,
-            corejs: 3,
-            modules: false,
-          },
-        ],
-      ],
-    })
-
-    const identifiers = Array.from(instance.state.identifiers)
-    const imports = Array.from(instance.state.imports)
-    if (!identifiers.includes(`React`)) {
-      identifiers.push(`React`)
-      imports.push(`import * as React from 'react'`)
-    }
 
-    results.scopeImports = imports
-    results.scopeIdentifiers = identifiers
-    // TODO: be more sophisticated about these replacements
-    results.body = result.code
-      .replace(
+    results.body = sucrase
+      .transform(code, { transforms: [`jsx`] })
+      .code.replace(
         /export\s*default\s*function\s*MDXContent\s*/,
         `return function MDXContent`
       )
       .replace(
         /export\s*{\s*MDXContent\s+as\s+default\s*};?/,
         `return MDXContent;`
       )
+      .replace(/^(\s*import\s+.*)/gm, `/*** $1 ***/`)
+      .replace(/^(\s*export\s+\{.*)/gm, `/*** $1 ***/`)
+      .replace(/^(\s*export\s+)(.*)/gm, `/*** $1 ***/ $2`)
```

Comment:
Note: these comments are mostly for debugging and this content can be dropped entirely.